numaprom/udf/window.py

```diff
@@ -33,37 +33,26 @@ def __aggregate_window(key, ts, value, win_size, buff_size, recreate) -> List[Tu
 @msg_forward
 def window(key: str, datum: Datum) -> Messages:
     msg = decode_msg(datum.value)
-    labels = msg.get("labels")
-    win_size = int(os.getenv("WIN_SIZE", DEFAULT_WIN_SIZE))
+    metric_name = msg["name"]
```

Comment:
The current decode_msg() implementation may return None upon json exception.
As we don't check if msg is None, we will be hiding the specific json exception in decode_msg() and trigger another / new exception.
Please fix this by either throwing in decode_msg or do None check here.

Reply:
@qhuai Thank you for catching this. None check should be there if decode_msg method is used. However, we do not through the error since its a stream and it would get stuck with the exception.  Also, decode_msg method is unnecessary, this should return None directly to the UDF. Made the fix.